### ISSUE TITLE
Diagram might be helpful to understand the layout of a figure

### ISSUE THREAD

**Issue by saptarshiguha** [mozilla/metrics-graphics#741]
A diagram describing the borders and extents of a figure might be useful to understand the definitions used in 

https://github.com/mozilla/metrics-graphics/wiki/Layout

thanks again
Saptarshi

**Comment by hamilton** [comment]
I agree. Now that I'm back and putting together a release, it's time to add some more documentation.
